manager/manager/typeresolver.go

```diff
@@ -129,32 +139,41 @@ func (tr *typeResolver) ResolveTypes(config *Configuration, imports []*ImportFil
 		}
 
 		if sch != "" {
-			var s Schema
+			var s common.Schema
 			if err := yaml.Unmarshal([]byte(sch), &s); err != nil {
 				return nil, resolverError(config, err)
 			}
 			// Here we handle any nested imports in the schema we've just fetched.
 			for _, v := range s.Imports {
-				i := &ImportFile{Name: v.Name}
+				i := &common.ImportFile{Name: v.Name}
 				var existingSchema string
-				if len(fetched[v.Path]) == 0 {
+				u, conversionErr := tr.MapFetchableURL(v.Path)
```

Comment:
So, did I break something, or is this something that already was broken. If this change is not making new breakages, then I'd like to fix this in a follow on cl.